src/content/docs/kubernetes-pixie/kubernetes-integration/installation/reduce-ingest.mdx

```diff
@@ -1,114 +1,284 @@
+
+```yaml
+common:
+  config:
+    namespaceSelector:
+      matchLabels:
+        key1 : "value1"
 ```
-HTTP_SPAN_LIMIT: 750
-DB_SPAN_LIMIT: 250
-COLLECT_INTERVAL_SEC: 15
+
+### Examples with namespaces [#include-namespaces]
+
+<CollapserGroup>
+    <Collapser
+        id="namespaces-label-true"
+        title={<>Scrape namespaces with namespaces with the label <InlineCode>newrelic.com/scrape</InlineCode> set to <InlineCode>true</InlineCode></>}
+    >
+        ```yaml
+        global:
+        licenseKey: _YOUR_NEW_RELIC_LICENSE_KEY_
+        cluster: _K8S_CLUSTER_NAME_
+
+        # ... Other settings 
+
+        # Configuration for newrelic-infrastructure
+        newrelic-infrastructure:
+        # ... Other settings 
+        common:
+            config:
+            namespaceSelector:
+                matchLabels:
+                newrelic.com/scrape: "true"
+        ```
+    </Collapser>
+    <Collapser
+        id="k8s-expressions"
+        title="Use Kubernetes expressions to include or exclude namespaces"
+    >
+    You can also use Kubernetes expressions to include or exclude namespaces using this syntax:
+
+    ```yaml
+    common:
+    config:
+        namespaceSelector:
+        matchExpressions:
+        - {key: newrelic.com/scrape, operator: NotIn, values: ["false"]}
+        - {key: key1, operator: In, values: ["value1"]}
+    ```
+
+    <Callout variant="tip">
+    The expressions under `matchExpressions` are concatenated.
+    </Callout>
+
+    </Collapser>
+    <Collapser
+        id="exclude-namespaces"
+        title={<>Exclude namespaces with the label <InlineCode>newrelic.com/scrape</InlineCode> set to <InlineCode>false</InlineCode></>}
+    >
+
+    ```yaml
+    global:
+    licenseKey: _YOUR_NEW_RELIC_LICENSE_KEY_
+    cluster: _K8S_CLUSTER_NAME_
+
+    # ... Other settings 
+
+    # Configuration for newrelic-infrastructure
+    newrelic-infrastructure:
+    # ... Other settings
+    common:
+        config:
+        namespaceSelector:
+            matchExpressions:
+            - {key: newrelic.com/scrape, operator: NotIn, values: ["false"]}
+    ```
+    </Collapser>
+</CollapserGroup>
+
+
+You can see a full list of settings that you can modify in the [chart's README file](https://github.com/newrelic/nri-kubernetes/tree/main/charts/newrelic-infrastructure).
+
+### How can I find out which namespaces are excluded? [#excluded-namespaces]
+
+The `K8sNamespace` sample shows all the namespaces within the cluster. The `nrFiltered` attribute determines whether we're going to scrape the data related to the namespace. This query  shows you which namespaces are being monitored:
+
+```sql
+FROM K8sNamespaceSample SELECT displayName, nrFiltered WHERE clusterName = <clusterName> SINCE 2 MINUTES AGO
```

Comment:
```suggestion
FROM K8sNamespaceSample 
SELECT displayName, nrFiltered 
WHERE clusterName = <clusterName> 
SINCE 2 MINUTES AGO
```

Reply:
Hey @rhetoric101, the query is fine in the preview and in the file. I don't know why you see the inserted spaces.